<launch>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    </include>
    <node pkg="gesture_detec_demo" type="GestureRos.py" name="GestureDetecDemo" output="screen" />
</launch>